### PR TITLE
Fixes #20594 - update qpid script for upgrades

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by ehelms** [comment]
@chris1984 Since #20239 has already been released, can you open a new issue?

File: hooks/pre/30-upgrade.rb [hunk 1]
```diff
@@ -119,24 +119,42 @@ def mark_qpid_cert_for_update
...
     Kafo::Helpers.execute("mkdir -p #{qpid_linearstore}/p001/efp/2048k/returned")
     Kafo::Helpers.execute("mkdir -p #{qpid_linearstore}/jrnl2")
-    # Backup data directory before upgrade
-    puts "Backing up #{qpid_dir} in case of migration failure"
-    Kafo::Helpers.execute("tar -czf /var/cache/qpid_queue_backup.tar.gz #{qpid_dir}")
-    # Move dat directory to new location dat2
-    Kafo::Helpers.execute("mv #{qpid_linearstore}/dat #{qpid_linearstore}/dat2")
+
+    if File.exist?("#{qpid_linearstore}/dat") && File.exist?("#{qpid_linearstore}/dat2")
+      Kafo::Helpers.execute("rm -f #{qpid_linearstore}/dat2")
```
Comment (ehelms) [inline hooks/pre/30-upgrade.rb]: `rm -rf`

File: hooks/pre/30-upgrade.rb [hunk 2]
```diff
@@ -145,12 +163,16 @@ def upgrade_qpid_paths
...
     # Restore access
     Kafo::Helpers.execute("chown -R qpidd:qpidd #{qpid_dir}")
+
     # restore SELinux context by current policy
     Kafo::Helpers.execute("restorecon -FvvR #{qpid_dir}")
     logger.info 'Qpid path upgrade complete'
-    Kafo::Helpers.execute("rm -f /var/cache/qpid_queue_backup.tar.gz")
+    Kafo::Helpers.execute("rm -f #{backup_file}")
+    logger.info 'Removing old jrnl directory'
+    Kafo::Helpers.execute("rm -rf #{qpid_linearstore}/jrnl")
```
Comment (ehelms) [inline hooks/pre/30-upgrade.rb]: This line will replace your other open PR ja?

Comment (chris1984) [inline hooks/pre/30-upgrade.rb]: Yeah it will, I closed the other one out

File: hooks/pre/30-upgrade.rb (hunk 1 above)
Comment (chris1984) [inline hooks/pre/30-upgrade.rb]: Thought I fixed that let me go back and refix it

**Comment by chris1984** [comment]
updated

**Comment by ehelms** [comment]
Mind updating the commit mesage to include mention of qpid?

**Comment by chris1984** [comment]
Updated the commit message

**Comment by chris1984** [comment]
@ehelms care if i merge? I updated the commit message to contain qpid.

**Pull request merged**